frontend/components/organization/Edit.jsx

```diff
@@ -108,6 +109,23 @@ function Edit({ actions }) {
 
   const getSandboxMsg = days => <span>{sandboxMsg(days)}</span>;
 
+  function getInvitationSentMsg() {
+    return (
+      <span>
+        The new member must create a Pages account to accept the invitation to this organization.
+        They may check their inbox at
+        {' '}
+        {mostRecentlyAddedUser.email}
+        {' '}
+        for an invitation by email or visit
+        {' '}
+        <a href="{mostRecentlyAddedUser.link}">{mostRecentlyAddedUser.link}</a>
```

Comment:
To use the template literal syntax you will want 
```
<a href={`${mostRecentlyAddedUser.link}`}>
```

Reply:
Do I? It's being interpolated as currently coded:
<img width="974" alt="image" src="https://github.com/cloud-gov/pages-core/assets/12150/5741e6b7-c4cd-496e-99f2-921ea191850b">

Reply:
The link/href itself is what needs to be changed

Reply:
_Ohhhhhh_ I see what you both mean. Gotcha; thanks for catching that and explaining!